### ISSUE TITLE
Creating order with take profit and stop loss in binance

### ISSUE THREAD

**Issue by kaanguven** [ccxt/ccxt#12533]
Hi,

I want to create future long or short order with take profit and stop loss .

For ex.

I created long contract and i wanna close it when profit = %1 AND loss = %1 so, what are the params,type etc.?

Thank you.

**Comment by ttodua** [comment]
late reply, but anyone with same problem: 
there are two different approaches you can use, please check 2nd and 3rd types: 
https://docs.ccxt.com/#/README?:~:text=We%20have%20different%20classification%20of%20trigger%20orders
then let us know if you can't clarify anything.
